lib/modules/platform/bitbucket-server/index.spec.ts

```diff
@@ -291,6 +291,117 @@ describe('modules/platform/bitbucket-server/index', () => {
           expect(httpMock.getTrace()).toMatchSnapshot();
         });
 
+        it('no git url', async () => {
```

Comment:
I think this test may not have been correctly resolved in the merge, I will investigate